src/Microsoft.HttpRepl.IntegrationTests/Commands/DeleteCommandsTests.cs

```diff
@@ -1,14 +1,19 @@
 using System.Threading.Tasks;
 using Microsoft.HttpRepl.Commands;
+using Microsoft.HttpRepl.FileSystem;
+using Microsoft.HttpRepl.IntegrationTests.Mocks;
 using Microsoft.HttpRepl.IntegrationTests.SampleApi;
 using Xunit;
 
 namespace Microsoft.HttpRepl.IntegrationTests.Commands
 {
     public class DeleteCommandTests : HttpCommandTests<DeleteCommand>, IClassFixture<HttpCommandsFixture<DeleteCommandsConfig>>
     {
+        private static readonly IFileSystem _fileSystem = new MockedFileSystem();
+
         private readonly DeleteCommandsConfig _config;
-        public DeleteCommandTests(HttpCommandsFixture<DeleteCommandsConfig> deleteCommandsFixture)
+        
+        public DeleteCommandTests(HttpCommandsFixture<DeleteCommandsConfig> deleteCommandsFixture) : base(new DeleteCommand(_fileSystem))
```

Comment:
nit: put `: base(...)` on the next line for readability

Reply:
Done.